microsoft-edge/web-platform/local-network-access.md

```diff
@@ -0,0 +1,119 @@
+
+An enterprise administrator can configure the Local Network Access policies to grant or deny a website access to an enterprise's devices.
+
+This feature requires Microsoft Edge 140 or later.
+
+
+<!-- ====================================================================== -->
+## What is Local Network Access?
+
+_Local Network Access_ is a feature in Microsoft Edge that restricts a website's ability to send requests to servers on a user's local network.  This feature displays a prompt to users (website visitors), requiring the user to grant or deny the website permission before these requests can be made.
+
+This feature delivers secure, privacy-respecting browsing experiences, while maintaining compatibility with modern web standards.
+
+The ability of a website to request this permission from users is restricted to secure contexts.
+
+
+<!-- ====================================================================== -->
+## How to Enable Local Network Access in Microsoft Edge
+
+Users (website visitors using Microsoft Edge) can enable and manage Local Network Access permissions through the Microsoft Edge browser settings.
+
+To grant or deny a website permission to access your local network:
+
+1. In Microsoft Edge, go to `edge://flags/#local-network-access-check`.
+
+1. In the dropdown list for **Local Network Access Checks**, select **Enabled (Blocking)**.
+
+   Or, select **Enabled**, which will make the website display a warning, rather than blocking the network request.
+
+1. Click the **Restart** button in the lower right.
+
+1. In Microsoft Edge, select **Settings and more** (![Settings and more icon](./local-network-access-images/settings-and-more-icon.png)) > **Settings**.
+
+   The **Settings** page opens.
+
+1. Select **Privacy, search, and services** > **Site permissions** > **All permissions**.
+
+   The **All permissions** page is displayed:
+
+   ![The All permissions page in Edge Settings](./local-network-access-images/settings-all-permissions.png) 
+
+1. Scroll down to the **Local network access** section, and then click it:
+
+   ![The Local Network Access section](./local-network-access-images/local-network-access-section.png)
+
+   The **Local network access** page is displayed, and the Address bar shows `edge://settings/privacy/sitePermissions/allSitePermissions/localNetworkAccess`:
+
+   ![Settings for the Local Network Access website permission](./local-network-access-images/settings-for-local-network-access.png)
+
+1. Turn on the **Ask before accessing (recommended)** toggle.
+
+
+<!-- ====================================================================== -->
+## Configuring policies
+
+Enterprise administrators can configure access on a per-site basis, by using the following policies:
+* [LocalNetworkAccessAllowedForUrls](/deployedge/microsoft-edge-browser-policies/localnetworkaccessallowedforurls)
```

Comment:
Update the two Policies Ref urls, to:
https://learn.microsoft.com/deployedge/microsoft-edge-browser-policies/localnetworkaccessallowedforurls
http://learn.microsoft.com/deployedge/microsoft-edge-browser-policies/localnetworkaccessblockedforurls

@murphys0709